inference-engine/tests/functional/plugin/shared/include/behavior/infer_request_cancellation.hpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <tuple>
+#include <vector>
+#include <string>
+#include <memory>
+#include <future>
+#include "ie_extension.h"
+#include <condition_variable>
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+#include "ngraph_functions/builders.hpp"
+#include <ie_core.hpp>
+#include <functional_test_utils/behavior_test_utils.hpp>
+#include "common_test_utils/common_utils.hpp"
+#include "functional_test_utils/plugin_cache.hpp"
+#include "functional_test_utils/blob_utils.hpp"
+#include "ngraph_functions/pass/convert_prc.hpp"
+#include "ngraph_functions/subgraph_builders.hpp"
+#include "behavior/infer_request_cancellation.hpp"
+
+namespace BehaviorTestsDefinitions {
```

Comment:
You also have to add test on canclelling the request executed with Infer() method

Reply:
Will do. BTW, why Infer do not has status while  StartAsync and Wait has?

Reply:
Done